### DISCUSSION TITLE
Org creation failed

### DISCUSSION THREAD

**Discussion by vinukumar-vs** [Sunbird-Lern/Community#90] [Issues] [answered]
While setting-up the Lern instance first time, we are facing the below error while creating the new org

**Response Error:**
{
    "id": "api.org.create",
    "ver": "v1",
    "ts": "2023-03-16 06:48:32:445+0000",
    "params": {
        "resmsgid": "d61826730cdd6e6e3d3aeb997f704515",
        "msgid": "d61826730cdd6e6e3d3aeb997f704515",
        "err": "UOS_ORGCRT0018",
        "status": "FAILED",
        "errmsg": "Invalid organisationType: board. Valid values are: []."
    },
    "responseCode": "CLIENT_ERROR",
    "result": {}
}

**Request**
Find the below CURL command used to create the org

curl --location '{{host}}/api/org/v1/create' \
--header 'Authorization: Bearer {{kong_api_key}}' \
--header 'x-authenticated-user-token:  {{kong_access_token}}' \
--header 'Content-Type: application/json' \
--data '{
    "request": {
        "orgName": "Sunbird Org",
        "channel": "sunbird",
        "description": "Default Organisation for Sunbird",
        "externalId": "sunbird",
        "isSSOEnabled": true,
        "organisationType": "board",
        "isTenant": true
    }
}

@reshmi-nair @Hari-stackroute

**Comment by vinukumar-vs** [comment]
I was referring to some reference to what is "organisationType". I found the below details
https://lern.sunbird.org/learn/product-and-developer-guide/user-and-org-service#organisation-type

But it is not explained what is the enum list that is supported. How can we define this.

@Krishnaj20 @rhwarrier @reshmi-nair @Hari-stackroute

**Comment by reshmi-nair** [comment, marked as answer]
@vinukumar-vs 
As discussed, pls set the default organisation types using system settings API. 
Sample data:
{
                "id": "orgTypeConfig",
                "field": "orgTypeConfig",
                "value": "{\"fields\":[{\"name\":\"School\",\"value\":2,\"description\":\"School\",\"displayName\":\"School\",\"flagNameList\":[\"isSchool\"]},{\"name\":\"Board\",\"value\":5,\"description\":\"Board\",\"displayName\":\"Board\",\"flagNameList\":[\"isBoard\"]}]}"
            }

**Reply by vinukumar-vs** [reply]
@reshmi-nair Can you please update the same in the related API documentation. So that it helps for everyone what is the dependent API for ORG creation.

@Krishnaj20 @rhwarrier  @Hari-stackroute

**Reply by rhwarrier** [reply]
@Hari-stackroute  pls update the documentation on the microsite as necessary and get back on this thread. Thanks.
